registry/README.md

```diff
@@ -4,37 +4,37 @@
 
 ### Build Package
 
-Create `azpkg` and upload to a ßstorage account.
-
-```powershell
-PackageGenerator\AzureGalleryPackager.exe -m <path-to-manifest.json> -o <output-dir>
-```
+Use the `make release` command to create `azpkg` file for Docker Container Registry Marketplace Item, and find it in the "_out" folder. Upload the file to a blob container in a storage account, and make sure to change the access policy of the blob to enable anonymous access.
 
 ### Side-load Package
 
-Connect to management endpoint and install package.
+Update admin credential and storage account information in the `gallery.ps1` script, and use it to connect to management endpoint and install package.
 
 ```powershell
 Scripts\gallery.ps1
 ```
 
 ## Solution Pre-requisites
 
-### Create Self-signed Certificate (optional)
+### Create Self-signed Certificate
+
+Update certificate information in the "self-signed.ps1" script, and use it to create a new certificate in `pfx` formatand and export it. If an existing certificate is preferred, use the `Export-PfxCertificate` command instead to export the certificate.
 
 ```powershell
 Scripts\self-signed.ps1
 ```
 
+This certificate will be uploaded to a key vault in a following step, and used by the private docker registry vm.
+
 ### Create Backend Storage and Key Vault
 
-Connect to tenant space
+Update the tenant information in the `connect.ps1` script, and usse it to connect to tenant space.
 
 ```powershell
 Scripts\connect.ps1
 ```
 
-Create backend storage and key vault
+Update instance, resources, certificate and docker registry user credential information in the "pre-reqs.ps1" script, and use it to create backend storage and key vault. The "Certificate URL" and "Certificate thumbprint" from script console output, together with the created storage account and key vault information, will be used in the follow step to create deployment for the private docker registry.
```

Comment:
- "Update variables *instance*, *resource*, ..."
- credentials, not credential
- casing: to the create THE backend Storage Account and Key Vault instance
- "from the script output"
- following, not follow
- "deploy the", not "create deployment for the"